### ISSUE TITLE
Fullscreen does not work as it should (Windows)

### ISSUE THREAD

**Issue by RayKoopa** [godotengine/godot#4599]
**Operating system or device - Godot version:**
Windows 10 Pro x64, Godot 2.0.2 official stable
AMD R9 390

**Issue description** (what happened, and what was expected):
Configuring the display settings to start the game in fullscreen creates a pretty weird output window.

- It has no borders (not really wrong).
- The top left corner is at Windows' default window position, not at 0,0 (thus, it wanders a bit every time I start the game, from close to top left to further down right).
- It extends over the screen borders, still having the desktop resolution's size.

Basically, it looks like this:
![image](https://cloud.githubusercontent.com/assets/9631152/15123792/90ebc946-1625-11e6-8b00-50dcd95517fe.png)

That game's internal resolution is 640x480. I think there are only two things wrong:

- The window must correctly get tucked to 0,0. Right now it looks like CW_DEFAULT is passed to the startup coordinates, IIRC.
- It must be above the taskbar.

**Comment by Calinou** [comment]
In my opinion, setting non-native fullscreen resolution should not be supported, for the following reasons:

- On almost all monitors, it'll cause mode setting which introduces a 2-second flicker.
- Using 4:3 resolutions on 16:9 makes no sense.
- There are better ways to create pixelated looks for games. One of them is G-buffer scaling, which can be set to only pixelate 2D/3D rendering, and not the fonts and GUI.

**Comment by RayKoopa** [comment]
It would not be pixelated anyway because graphics cards apply some filtering too. I wanted to do it because of mouse movement speed. If i move the mouse from left to right at 640 width it's fast, but when i maximize the window, it gets slower because it now spans 1080 pixels. I might need another input method for that though, as that effect is a bit annoying in a break out style game.

**Comment by reduz** [comment]
This is not a bug, you can use Viewport scale mode to keep your desired resolution but still make it span to fullscreen

**Comment by RayKoopa** [comment]
I don't know how that applies to the bad window positioning Godot does?

**Comment by vnen** [comment]
I'm pretty sure this was fixed by #5894.
